aspnetcore/blazor/components.md

```diff
@@ -83,6 +85,8 @@ The following markup in *Index.razor* renders a `HeadingComponent` instance:
 
 [!code-cshtml[](common/samples/3.x/BlazorSample/Components/HeadingComponent.razor)]
 
+If a component contains an HTML element with an uppercase first letter that doesn't match a component name, a warning is emitted indicating that an element has an unexpected name&mdash;adding an `@using` statement for the component's namespace overrides the warning.
```

Comment:
> adding an `@using` statement for the component's namespace overrides the warning.

This isn't really accurate. The warning points out that you attempted to bind a component, but that component wasn't found. Adding a using isn't a way to *suppress* the warning, it fixes the underlying problem (component was not found).